src/dali/plugin/input/rest/binance_com.py

```diff
@@ -1125,10 +1125,14 @@ def _process_gain(self, transaction: Any, transaction_type: Keyword, in_transact
             amount = RP2Decimal(transaction[_AMOUNT])
             notes = f"{notes + '; ' if notes else ''}{transaction[_EN_INFO]}"
 
+            # Just in case binance changes it's format in the future, let's check if it looks like a transaction hash
+            if re.search(r"^([A-Fa-f0-9]{64})$", transaction[_ID]):
```

Comment:
I'm not sure this check is valid for all altcoins.

Reply:
Yeah I figure it's not completely accurate, but better than nothing. Is there a canonical place where a regex to identify transaction hashes on all networks exist?

Reply:
This may produce incorrect results: e.g. if there is an altcoin with transaction hashes that are not 64-byte long, then this logic would skip it. I don't think there is a way to validate transaction hashes for all possible coins, at least AFAIK.

Reply:
Agreed! If we were actually doing something with the IDs (i.e. including them as uids) then I'd be more worried, but since it's just a helpful log for the user I'm thinking this is fine.

Reply:
Oh, I misinterpreted the code: you're just generating an info for the user. Got it, thanks for the explanation.